sentry-core/src/main/java/io/sentry/core/protocol/SentryId.java

```diff
@@ -27,6 +28,19 @@ public String toString() {
     return uuid.toString().replace("-", "");
   }
 
+  @Override
+  public boolean equals(Object o) {
```

Comment:
I think you can use `compareTo(UUID val)` and you don't need to override anything

Reply:
I'll use it internally to compare the UUIDs, but `SentryId` I belive should override it. One might accidentally use equals here (as I did). I'd say it's probably the only type worth overriding so far.

Will do as you suggested: `return uuid.compareTo(sentryId.uuid) == 0;`